deploy/kubernetes/che-plugin-registry/templates/deployment.yaml

```diff
@@ -56,3 +56,7 @@ spec:
             memory: {{ .Values.chePluginRegistryMemoryLimit }}
           requests:
             memory: 256Mi
+        envFrom:
+        - configMapRef:
+            name: che-devfile-registry
```

Comment:
`che-devfile-registry` ? did you mean `che-plugin-registry` ?

Reply:
Good catch, copy paste error :)

Fixed.